docs/blog/newsletter-june-2020.rst

```diff
@@ -0,0 +1,132 @@
+Common words and how to identify them
+-------------------------------------
+
+When we decide which words to use in our documentation, it's important to determine how common a word is. Common words are easier to understand and make your documentation more accessible to a wider audience. Picking the more common word can also be a handy way to settle a word choice disagreement - as enjoyable as the debate may be!
+
+In addition to researching the words you use, you can user test them. If your (representative) test users don't know what the words in a document mean, or have difficulty understanding the procedure because of the words used, this can help you confirm that the words are uncommon, or at least not right for your audience.
+
+Here are some tools to help you tell if a word is common or uncommon:
+
+*   `Google Ngram <https://books.google.com/ngrams>`__
+*   `Google Trends <https://trends.google.com/trends/?geo=US>`__
+*   `iWeb Corpus <https://www.english-corpora.org/iweb/>`__
+*   `1000 Most Common English Words <https://1000mostcommonwords.com/1000-most-common-english-words/>`__
+*   `Plain English Campaign (guidelines) <http://www.plainenglish.co.uk/>`__
+*   `Simplified Technical English (guidelines) <https://en.wikipedia.org/wiki/Simplified_Technical_English>`__
+
+
+-------------------------------------------
+Obvious things and whether to document them
+-------------------------------------------
+
+Recently, a discussion on Slack was sparked on the topic of whether or not you should document the “obvious”.
+
+Those against documenting the obvious gave the example of uncomplicated UI procedures like deleting an object: if it’s something you’d expect users of almost any software to be familiar with, you could expect your audience to know it already. Plus, the downsides of being explicit include describing how to fill in every field in detail, or multi-step procedures for something the UI guides you through. More detail like this can make docs feel very long and cluttered, and make it hard for readers to find the bit that’s important to them. And explaining things in too much depth can come across as condescending.
+
+In favour of documenting the obvious: what’s obvious to you isn’t necessarily obvious to others. Besides, those who already know something can skip over it. This discussion provides a reminder of how important it is to know your user base - what is "obvious and familiar" depends on your background, exposure, and contexts. At minimum, make sure you know what your assumptions are: if your assumptions are explicit, you can make deliberate choices on what to leave in and out, based on the intended audience.
+
+Ultimately, be guided by what your specific audience needs to know to achieve their goals. That’s what belongs in documentation.
+
+-------------------------------------------------------------------------------------
+From our sponsor: Open Source Curious? Learn (and earn) with Google's Season of Docs!
```

Comment:
I feel like I must have a special attraction to hyphens this month... "Open Source-Curious?" maybe?